install/kubernetes/helm/istio/charts/grafana/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v1
 description: A Helm chart for Kubernetes
 name: grafana
 version: 1.0.1
-appVersion: 1.0.1
+appVersion: 5.2.2
```

Comment:
Should not this be 1.0.1? I think Istio is using its own grafana?

Reply:
appVersion is the version of the application in use

version is the version of the chart.

I guess it is open to interpretation...  If you can find any docs that contradict this approach lmk :)